src/ipelib/IPE_Model_Class.F90

```diff
@@ -213,7 +213,7 @@ SUBROUTINE Update_IPE_Model( ipe, t0, t1, rc )
       call ipe % forcing % Update_Current_Index( ipe % parameters, &
                                                  ipe % mpi_layer, &
                                                  ipe % forcing_io, &
-                                                 ipe % time_tracker % elapsed_sec, &
+                                                 ipe % time_tracker % elapsed_sec - ipe % forcing % start_time, &
```

Comment:
This could be done internally in `Update_Current_Index()` since `start_time` is defined inside `IPE_Forcing`.